File: DSA/LAB 2/Rectangle.c
int rectangle()
{

    int l1, b1, l2, b2, l3, b3;

    printf("Enter length and breadth of rectangle 1: ");
    scanf("%d%d", &l1, &b1);

    printf("Enter length and breadth of rectangle 2: ");
    scanf("%d%d", &l2, &b2);

    printf("Enter length and breadth of rectangle 3: ");
    scanf("%d%d", &l3, &b3);

    int p1 = 2 * (l1 + b1);
    int p2 = 2 * (l2 + b2);
    int p3 = 2 * (l3 + b3);

    printf("%d : %d : %d", p1, p2, p3);

    p1 > p2 &&p1 > p3 ? printf("\nRectangle 1 has the greatest perimeter") : (p2 > p3 && p2 > p3 ? printf("\nRectangle 2 has the greatest perimeter") : (p3 > p2 && p3 > p1 ? printf("\nRectangle 3 has the greatest perimeter") : printf("No rectangle")));

    return 0;
}

File: Python/Experiment 7/rectangle.py
class Rectangle:
    def __init__(self, length, breadth):
        self.length = length
        self.breadth = breadth
    
    def area(self):
        return self.length*self.breadth;

    def perimeter(self):
        return 2*(self.length + self.breadth)


length = int(input('Please enter length of the rectangle : '))
breadth = int(input('Please enter breadth of the rectangle : '))

r1 = Rectangle(length,breadth)


print('The length and breadth of the rectangle is {} {}'.format(r1.length, r1.breadth))
print('Area of the rectangle is {}'.format(r1.area()))
print('Perimeter of the rectangle is {}'.format(r1.perimeter()))

